electronics/resonant_frequency.py

```diff
@@ -0,0 +1,41 @@
+# https://en.wikipedia.org/wiki/LC_circuit
+from __future__ import annotations
+
+from math import pi, sqrt
+
+
+def resonant_frequency(inductance: float, capacitance: float) -> tuple:
+    """
+    This function can calculate the resonant frequency of LC circuit,
+    for the given value of inductance and capacitnace.
+
+    Examples are given below:
+    >>> resonant_frequency(inductance=0, capacitance=5)
+    Traceback (most recent call last):
+      ...
+    ValueError: Inductance cannot be 0 or negative
+    >>> resonant_frequency(inductance=10, capacitance=0)
+    Traceback (most recent call last):
+      ...
+    ValueError: Capacitance cannot be 0 or negative
+    >>> resonant_frequency(inductance=10, capacitance=5)
+    ('Resonant frequency', 0.022507907903927652)
```

Comment:
These doctests are good, however following general conventions, maybe you can move the successful case/s before the failed ones.

Reply:
I have updated the doctests according to the conventions.